tests/unit/broccoli/default-packager/tests-test.js

```diff
@@ -208,6 +208,8 @@ describe('Default Packager: Tests', function() {
     expect(outputFiles.assets['tests.js']).to.include(`define('the-best-app-ever/config/environment'`);
     expect(outputFiles.assets['tests.js']).to.include(`require('the-best-app-ever/tests/test-helper');`);
     expect(outputFiles.assets['tests.js']).to.include('EmberENV.TESTS_FILE_LOADED = true;');
+
+    expect(defaultPackager.options.fingerprint.exclude).to.include('testem');
```

Comment:
it is a little confusing to have this assertion here; the code above clearly passes a fingerprint object with empty `exclude` array.

I think it would be better to add a separate test

```javascript
it('excludes testem files by default', co.wrap(function *() {
  let defaultPackager = new DefaultPackager({
    // ... code ...
    fingerprint: {
      exclude: []
    }
  });

  output = yield buildOutput(defaultPackager.packageTests(input.path()));

  expect(defaultPackager.fingerprint.exclude).to.include('testem');
});
```

But as I mentioned in the comment, I think `exclude` array should be passed in from the `ember-app`. That would mean moving [this code](https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/default-packager.js#L1125-L1127) to `ember-app` and just using the argument within the packager itself:

```javascript
// ... code ...
if (this.fingerprint && this.fingerprint.exclude) {
  this.fingerprint.exclude.push('testem');
}

this._defaultPackager = new DefaultPackager({
  // ... code ...
  fingerprint: this.fingerprint,
});
```